scripts/install-operator-sdk.sh

```diff
@@ -14,21 +14,29 @@ set -eo pipefail
 
 SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 ROOT_DIR="$SCRIPTS_DIR/.."
-DEFAULT_OPERATOR_SDK_VERSION="1.7.1"
+DEFAULT_OPERATOR_SDK_BIN_PATH="$ROOT_DIR/../../aws-controllers-k8s/code-generator/bin"
```

Comment:
```suggestion
DEFAULT_OPERATOR_SDK_BIN_PATH="$ROOT_DIR/../code-generator/bin"
```

Reply:
In addition to the above, does `OPERATOR_SDK_BIN_PATH` become a configurable at this point?

If that's the goal, then :+1:. But you might want to update the documentation at the top of this script to indicate that this is a configurable.

Reply:
I see. Can we update the other script to do the same, then?

Reply:
@RedbackThomson I can update the other script, but I'd prefer that be a separate PR.